src/index.html

```diff
@@ -22,6 +22,373 @@
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="Product photo"
+          class="card__image"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__rate">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__counter">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-title">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+        <a
+          href="/"
+          class="card__buy-button"
+          data-qa="hover"
+        >
+          Buy
+        </a>
+      </article>
+
+      <article
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="Product photo"
+          class="card__image"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__rate">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__counter">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-title">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+        <a
+          href="/"
+          class="card__buy-button"
+          data-qa="hover"
+        >
+          Buy
+        </a>
+      </article>
+
+      <article
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="Product photo"
+          class="card__image"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__rate">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__counter">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-title">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+        <a
+          href="/"
```

Comment:
The `href` attribute for the 'Buy' button should lead to a product page or be a placeholder ('#'), not the root path '/'.